src/mc.rs

```diff
@@ -178,7 +180,7 @@ unsafe fn put_8tap_rust<BD: BitDepth>(
             mid_ptr = &mut mid[128 * 3..];
             for _ in 0..h {
                 for x in 0..w {
-                    dst[x] = rav1d_filter_8tap_clip(
+                    *dst.add(x) = rav1d_filter_8tap_clip(
```

Comment:
Can you do these other `dst.add(x)` ones, too?

Reply:
Done.